src/Components/Reports/Util.js

```diff
@@ -1,6 +1,13 @@
 import { get } from 'lodash';
+import moment from 'moment';
 import { pdfRowKeys, percentageKeys, reportRowKeys, SYSTEMS_REPORT_FILE_NAME } from '../../constants';
 
+export const formatLastReported = (date) => {
+    const today = moment().format('YYYY-MM-DD');
+    const isTodayWithoutTime =  today === date;
+    return isTodayWithoutTime ? 'Today' : moment(date).fromNow();
```

Comment:
Reference for using this mothod: https://momentjscom.readthedocs.io/en/latest/moment/04-displaying/02-fromnow/